src/app/user-form/user-form.component.css

```diff
@@ -0,0 +1,65 @@
+.bvn{
```

Comment:
rename classes

Reply:
I agree with that. We can rename this class to .user-form-header